app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  after_create :update_comments_counter
+  belongs_to :post
+  belongs_to :author, class_name: 'User'
```

Comment:
- [Optional] Please as the last reviewer said you should add `private` before defining the rails active callbacks, also please it would be better to use after_save instead of after_create because you want to update the counter only if the comment was saved